static/templates/user_info_popover_content.hbs

```diff
@@ -52,31 +52,24 @@
     </div>
 
     {{#if is_me}}
-        <hr>
-        {{#if can_set_away}}
-        <li>
-            <a href="#" class="set_away_status">
-                <i class="fa fa-minus-circle" aria-hidden="true"></i> {{#tr this}}Set yourself as unavailable{{/tr}}
-            </a>
-        </li>
-        {{/if}}
-        {{#if can_revoke_away}}
-        <li>
-            <a href="#" class="revoke_away_status">
-                <i class="fa fa-minus-circle" aria-hidden="true"></i> {{#tr this}}Set yourself as active{{/tr}}
-            </a>
-        </li>
-        {{/if}}
-        <li>
-            <a href="#" class="update_status_text">
-                <i class="fa fa-comments" aria-hidden="true"></i>
-                {{#if status_text}}
-                    {{#tr this}}Edit status message{{/tr}}
-                {{else}}
-                    {{#tr this}}Set a status message{{/tr}}
-                {{/if}}
-            </a>
-        </li>
+    <hr>
+    <label for="status_info" class="dropdown-title">{{t "Status:" }}
+        <select name="status_info" class="status_info" id="status_info" >
+            {{#each status_info_values}}
+            <option value='{{this.value}}'>{{this.description}}</option>
```

Comment:
As we discussed, you need to define the selected option here based on the current user status so we can detect when it changes, making this works when a user changes their status over and over again.